### ISSUE TITLE
[Enhancement] XMB Additions

### ISSUE THREAD

**Issue by shinra358** [libretro/RetroArch#7878]
I would like to request addition sony xmb wave animations. Retroarch already uses the first iteration of the playstation 3 wave background animations. I would like the ask for the PSP, PS3 second iteration (w/sparkles), and the PS4 wave versions.

Along with that, I would like to ask for text/menu fade disappearing options so that if a user is on the menu for a while, after a certain time period of inactivity (controlled by a menu option), all text would disappear fade only leaving the wave background showing. All text would come back on keyboard or gamepad button/direction press and it would fade transition back in. This visual only feature would have options for it to be optional.

Basically it would be like an in-menu retroarch screensaver. An option could also be applied for if a user is in-game, it could be used as a screensaver if there is no input activity and it would fade transition in or back just like it would in the menu.
